glean-core/android/src/main/java/mozilla/telemetry/glean/private/CommonMetricData.kt

```diff
@@ -5,7 +5,6 @@
 package mozilla.telemetry.glean.private
 
 import mozilla.telemetry.glean.Glean
-// import mozilla.components.support.base.log.logger.Logger
 
 /**
  * Enumeration of different metric lifetimes.
```

Comment:
With `CommonMetricData` gone, we should consider renaming this file `Lifetime.kt` (not in this PR, but let's add a TODO if you agree).